src/apps/vpn/inspector/inspectorhandler.cpp

```diff
@@ -146,6 +148,16 @@ static QList<InspectorCommand> s_commands{
                        MozillaVPN::instance()->controller()->quit();
                        return QJsonObject();
                      }},
+    InspectorCommand{"live_reload", "Live reload file X", 1,
+                     [](InspectorHandler*, const QList<QByteArray>& args) {
+                       if (s_hotreloader == nullptr) {
+                         s_hotreloader = new InspectorHotreloader(
```

Comment:
In this way you create multiple obj for the same file, right?

Reply:
We always only update the path, not create new inserts.